db/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
If you want add a README.md, you should include more information, otherwise remove it